fhirr4/ballerina/src/main/resources/fhirservice/http_service_builder.bal

```diff
@@ -156,6 +156,12 @@ isolated function getHttpService(Holder h, r4:ResourceAPIConfig apiConfig, strin
 
         isolated resource function post [string... path](http:Request req, http:RequestContext ctx) returns any|error {
 
+            map<string>|http:ClientError formParams = req.getFormParams();
```

Comment:
Don't we need to set these params to fhirContext ?

Reply:
it's set in the later stage as search parameters, since it's set as request params: https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/pull/434/files#diff-68c8b9730aa0fe839a8b73e750a895279809643685eba45480a94fbcd8322211R596